pkg/modules/authorization.go

```diff
@@ -714,6 +714,7 @@ func authorizationStorageServiceV2(ctx context.Context, isDeleting bool, cr csmv
 		return fmt.Errorf("applying/deleting vault certificates: %w", err)
 	}
 
+	// replicates needs to be a int32
```

Comment:
Misspelling on replicas

Reply:
resolved, please feel free to use the suggestions with the comments to provide a fix